### ISSUE TITLE
Support abs file path

### ISSUE THREAD

**Issue by konosubakonoakua** [jesseduffield/lazygit#3253]
**Is your feature request related to a problem? Please describe.**
Currently I'm using key mapping below to open file in neovim:
```yaml
  - key: 'o'
    command: 'nvim --server $NVIM --remote-tab "$(git -C $(dirname {{.SelectedFile.Name}}) rev-parse --show-toplevel)/{{.SelectedFile.Name}}"'
    context: 'files'
    subprocess: true
    description: 'Open file in Neovim'

```
It is kind of ugly workaround, can we have a more elegant method?

**Describe the solution you'd like**
It would be nice if we can use `abspath`:
```yaml
  - key: 'o'
    command: 'nvim --server $NVIM --remote-tab {{.SelectedFile.abspath}}"'
```

**Related**
https://github.com/jesseduffield/lazygit/discussions/3160

**Comment by stefanhaller** [comment]
You don't need a custom command for that at all. Set your editor preset to `nvim-remote`, and press 'e' instead of 'o'.

```
os:
  editPreset: "nvim-remote"
```

**Comment by konosubakonoakua** [comment]
Thanks for your advice. But, do we must rely on a external tool to acheive?
I have tried `nvim-remote` & the `--pipe` way, Simply works but the former may not suitable for some environments that don't have python env. As for the later, need to make a alias in shell profile.

**Comment by stefanhaller** [comment]
I don't understand what you're talking about. There is no external tool involved, editor presets are built in to lazygit.

**Comment by konosubakonoakua** [comment]
I misunderstood the `neovim-remote`, there's a python tool named [neovim-remote](https://github.com/mhinz/neovim-remote).

BTW, I tried the `editPreset: "nvim-remote"`, I can do editing in the same floating terminal which hosts lazygit, but it seems that neovim don't load user config files just like `neovim -u NONE`, also, there's a annoying drawback for me. 😭 

I frequently change 'cwd' in neovim, the `editPreset` wont work after the `cwd` changed because it seems like use `relative path` too.

**Comment by stefanhaller** [comment]
> BTW, I tried the `editPreset: "nvim-remote"`, I can do editing in the same floating terminal which hosts lazygit, but it seems that neovim don't load user config files just like `neovim -u NONE`, also, there's a annoying drawback for me. 😭

I have zero experience with neovim, so I can't say anything about this. However, I don't understand why this is a problem with our editPreset but not with your custom command, as both seem to be using the exact same command.

> I frequently change 'cwd' in neovim, the `editPreset` wont work after the `cwd` changed because it seems like use `relative path` too.

This is not our fault, it's a bug in neovim, see https://github.com/neovim/neovim/issues/18519. Of course, we could easily pass absolute paths to all edit commands to work around it; this should help with this issue, and I don't see any downside for other editors. See #3255.

**Comment by konosubakonoakua** [comment]
> > BTW, I tried the `editPreset: "nvim-remote"`, I can do editing in the same floating terminal which hosts lazygit, but it seems that neovim don't load user config files just like `neovim -u NONE`, also, there's a annoying drawback for me. 😭
> 
> I have zero experience with neovim, so I can't say anything about this. However, I don't understand why this is a problem with our editPreset but not with your custom command, as both seem to be using the exact same command.
> 
> > I frequently change 'cwd' in neovim, the `editPreset` wont work after the `cwd` changed because it seems like use `relative path` too.
> 
> This is not our fault, it's a bug in neovim, see [neovim/neovim#18519](https://github.com/neovim/neovim/issues/18519). Of course, we could easily pass absolute paths to all edit commands to work around it; this should help with this issue, and I don't see any downside for other editors. See #3255.

Thank you for the answer and effort on #3255, hope it will be merged soon :)

**Comment by carbon-steel** [comment]
I have just come across this very issue (also in setting up neovim edit commands) and am amazed to find that the fix was merged in just 4 days ago (in https://github.com/jesseduffield/lazygit/pull/3255). Thanks so much for the fix :)

**Comment by stefanhaller** [comment]
You're welcome. Thanks for reminding me to close this issue. 😄

**Comment by stefanhaller** [comment]
Fixed by #3255.
